arrow-core-data/src/main/kotlin/arrow/unsafe.kt

```diff
@@ -1,33 +0,0 @@
-package arrow
-
-import arrow.typeclasses.Continuation
-import kotlinx.atomicfu.AtomicRef
-import kotlinx.atomicfu.atomic
-import kotlin.coroutines.CoroutineContext
-import kotlin.coroutines.EmptyCoroutineContext
-import kotlin.coroutines.RestrictsSuspension
-import kotlin.coroutines.startCoroutine
-
-private class UnsafeContinuation<A> : Continuation<A> {
-  val result: AtomicRef<A?> = atomic(null)
-
-  override fun resume(value: A) {
-    result.value = value
-  }
-
-  override fun resumeWithException(exception: Throwable) {
-    throw exception
-  }
-
-  override val context: CoroutineContext = EmptyCoroutineContext
-}
-
-@RestrictsSuspension
-object unsafe {
```

Comment:
This would need a deprecation cycle, to not break dependent API's like in Arrow Fx. Unless we want to delete it from core and move it to Fx. 
So far only fx, fx-rx2 need this operator 
cc @nomisRev , @raulraja

Reply:
This should go through a deprecation cycle in core directly.